lib/Elastica/Query/ParentId.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace Elastica\Query;
+
+final class ParentId extends AbstractQuery
```

Comment:
I would prefer to remove it as we don't have it in any other files and I don't see a reason, why we should prevent someone extendings this class if we wants to (even though it is unlikely to happen).